src/glossary.rst

```diff
@@ -1216,102 +1449,133 @@ Glossary
 
-         Let's say the workflow has run to cycle 8, but we have just noticed that
-         something went wrong with the task ``a`` in cycle 5.
+      Let's say the workflow has run to cycle 8, but we have just noticed that
+      a corrupted ancillary file resulted in bad products at cycle 5.
 
-         To rectify this we could create a new flow (a reflow) starting at ``a.5``::
+      To rectify this we could fix the corrupted file and trigger a new flow
+      (a reflow) from ``proc.5``::
 
-            # get the ID of the original flow
-            cylc TBC <id> a.5  # ???
+         cylc trigger a.5 --reflow <workflow-id>
 
-            # pause the original flow
-            cylc pause <id> --flow <flow>
+      The new flow will regenerate and republish the cycle 5 products before
+      naturally coming to a halt because the the triggered tasks do not lead on
+      to the next cycle and the rest of the graph.
 
-            # create a new flow starting at a.5
-            cylc trigger a.5 --reflow
+      Meanwhile, the original flow will carry on unaffected, from cycle 8.
 
-         Where ``<id>`` is the name of the :term:`workflow` and ``<flow>`` is the
-         identifier of the :term:`flow`.
+      If a task in one flow catches up to an active sibling from another flow
+      (same name and :term:`cycle point`, different flow number) they will
+      merge and carry both flow numbers forward. Downstream tasks can be
+      considered to belong to both flows.
+
+      Individual flows can be stopped with ``cylc stop --flow=<flow-number>``.
+      This removes the target flow number from all active tasks. If a task has
+      no flow numbers left it will not spawn downstream, thus stopping the flow.
+     
+      .. note::
+         Currently you have to look at the :term:`scheduler log` to identify
+         the flow numbers of particular tasks in the :term:`n=0 window <n-window>`.
 
-         This new flow will catch up and merge with the old one, the workflow
-         will then continue as normal.
 
-         Now we have started the new flow we can stop the original one::
+   event
+      An event is a milestone in the lifecycle of a :term:`workflow` or
+      :term:`task` at which the :term:`scheduler` provides a hook for
+      attaching :term:`event handlers <event handler>`.
 
-            # stop the original flow
-            cylc stop <id> --flow <flow>
+      Workflow events include :term:`startup`, :term:`stall`, and
+      :term:`shutdown`.
 
-         The new flow will now continue to run to the end.
+      Task events include :term:`task state` changes, to ``running`` or
+      ``failed``, for example, or when the scheduler receivers CRITICAL or
+      WARNING messages from a task :term:`job`.
 
-         It is also possible to have multiple :term:`flows <flow>` running in the
-         scheduler :term:`scheduler` simultaneously.
 
+   .. cylc-flow workflow cfgspec current reference "event handlers" plural
```

Comment:
What's this?

Reply:
I've made the comment clearer. Glossary terms are all singular except this one, which can't be fixed without a cylc-flow change.